Magical-Kitties/Magical-Kitties-Save-the-Day.css

```diff
@@ -162,196 +169,178 @@ h1, h2, h3, h4, h5, h6, .redtext, .pinktext, .whiteonredtext {
-}
-
-.flex-end {
-  justify-content: flex-end;
-}
-
-.flex-even {
-  justify-content: space-evenly;
+.rightalign {
+  text-align: right;
 }
 
-.flex-start {
-  justify-content: flex-start;
-}
+/* Grid displays */
 
-.flex-grow-2 {
-  flex-grow: 2;
+.grid-grow-2 {
+  grid-column: 5 / 13;
+  width: auto;
 }
 
-textarea.flex-grow-2 {
+textarea.grid-grow-2 {
   height: 15px;
 }
 
-
 /* Width-related formatting */
 
-.width100 {
-  width: 100px;
-  max-width: 100px;
-}
-
 .width200 {
-  width: 200px;
-  max-width: 200px;
-}
-
-.twentywide {
-  display: inline-block;
-  max-width: 18%;
-  width: 18%;
+  grid-column: 1 / 5;
 }
 
-.quarterwide {
-  display: inline-block;
-  max-width: 23%;
-  width: 23%;
+div.halfwide {
+  grid-column: 8 / 13;
+  margin: 0;
+  padding: 0;
 }
 
-.thirdwide {
-  display: inline-block;
-  max-width: 33%;
-  width: 33%;
+.fullwide {
+  grid-template-columns: repeat(6, 1fr);
+  grid-column: 1 / 13;
 }
 
-.fortywide {
-  display: inline-block;
-  max-width: 37%;
-  width: 37%;
+.roll-talent-powers {
+  grid-column: 5 / 13;
 }
 
-.halfwide {
-  display: inline-block;
-  max-width: 46%;
-  width: 46%;
-}
+/* Miscellaneous formatting */
 
-.threequarterswide {
-  display: inline-block;
-  max-width: 80%;
-  width: 80%;
+div.trackables {
+  display: grid;
+  grid-column: 1 / 8;
+  gap: inherit;
+  margin: 0;
+  padding: 0;
 }
 
-.fullwide {
-  display: inline-block;
-  max-width: 96%;
-  width: 96%;
+.trackables-list {
+  grid-column: 1 / 3;
 }
 
-
-/* Miscellaneous formatting */
-
-.inline-block {
-  display: inline-block;
+.trackables-limits {
+  grid-column: 3;
 }
 
 .no-display {
   display: none;
 }
 
-input[type="text"].input-large{
-   font-size: 24pt; 
-   justify-content: left;
-   align-items: left;
-   text-align: left;
+input[type="text"].input-large {
+  font-size: 24pt;
+  justify-content: left;
+  align-items: left;
+  text-align: left;
 }
 
-input[type="number"].input-large {
-   font-size: 24pt; 
-   justify-content: right;
-   align-items: right;
-   text-align: right;
+.trackables input[type="number"].input-large {
+  font-size: 24pt;
+  text-align: right;
+  height: 5rem;
+  width: 8rem;
+  margin-left: auto;
 }
 
 input[type="text"].input-medium {
-   font-size: 18pt; 
-   justify-content: left;
-   align-items: left;
-   text-align: left;
+  font-size: 18pt;
+  justify-content: left;
+  align-items: left;
+  text-align: left;
 }
 
 input[type="number"].input-medium {
-   font-size: 18pt; 
-   justify-content: right;
-   align-items: right;
-   text-align: right;
-}
-
-input[type="number"].46h{
-  height: 46.5px;
+  font-size: 18pt;
   justify-content: right;
   align-items: right;
   text-align: right;
 }
 
-textarea.kitty-desc{
+.trackables input[type="number"].limit {
+  margin-left: unset;
+  margin-right: auto;
+}
+
+textarea.kitty-desc {
   min-height: 260px;
   font-size: 18pt;
-  margin: 5px 0px;
+  margin: 5px 0 0 0;
+  width: 96.5%;
+}
+
+textarea.fullwide {
+  width: 98.5%;
```

Comment:
In Google Chrome, the scrollbar on textarea elements seems to make it slightly wider than the actual element, so shrink 'em down a little